app/src/marketplace/components/ProductPage/ProductOverview/index.jsx

```diff
@@ -0,0 +1,180 @@
+// @flow
+
+import React, { useState, useEffect, useCallback } from 'react'
+import cx from 'classnames'
+
+import type { Product } from '$mp/flowtype/product-types'
+import type { ResourceKeyId } from '$shared/flowtype/resource-key-types'
+import DeploySpinner from '$shared/components/DeploySpinner'
+import { getCommunityStats } from '$mp/modules/communityProduct/services'
+import useInterval from '$shared/hooks/useInterval'
+import useIsMounted from '$shared/hooks/useIsMounted'
+import DonutChart from '$shared/components/DonutChart'
+import Dropdown from '$shared/components/Dropdown'
+import MembersGraph from '../MembersGraph'
+
+import styles from './productOverview.pcss'
+
+type Props = {
+    product: Product,
+    authApiKeyId?: ?ResourceKeyId,
+    adminFee: number,
+    subscriberCount: number,
+    className?: string,
+}
+
+const CP_SERVER_POLL_INTERVAL_MS = 10000
+
+const ProductOverview = ({
+    product,
+    authApiKeyId,
+    adminFee,
+    subscriberCount,
+    className,
+}: Props) => {
+    const isMounted = useIsMounted()
+    const [isDeploying, setIsDeploying] = useState(true)
+    const [stats, setStats] = useState(null)
+    const [shownDays, setShownDays] = useState(7)
+
+    const getStats = useCallback(async () => {
+        try {
+            setIsDeploying(true)
+            const result = await getCommunityStats(product.beneficiaryAddress)
+            if (!isMounted) {
+                return
+            }
+            result.totalEarnings = 100
+            result.memberCount = {
+                total: 100,
+                active: 80,
+                inactive: 20,
+            }
+            setStats(result)
+            setIsDeploying(false)
+        } catch (e) {
+            console.error(e)
+        }
+    }, [product, isMounted])
+
+    useInterval(() => {
+        if (isDeploying) {
+            getStats()
+        }
+    }, CP_SERVER_POLL_INTERVAL_MS)
+
+    useEffect(() => {
+        getStats()
+    }, [getStats])
+
+    const productAgeMs = Date.now() - new Date(product.created || 0).getTime()
```

Comment:
uh, note that `new Date(0)` is `Jan 01 1970`, if there's no created date, perhaps `Date.now() - 1000` or something  would be a better default, so that it doesn't think the product is 50 years old? Though maybe a crazy date would reveal bugs more easily?

Reply:
I was thinking the latter option to reveal bugs more easily 👍